remote-trait-object/src/transport.rs

```diff
@@ -21,7 +21,7 @@ pub enum SendError {
     Closed,
 }
 
-pub trait TransportSend: Send {
+pub trait TransportSend: Sync + Send + std::fmt::Debug {
```

Comment:
How did you conclud that we can add `Send` in the `TransportSend`?

Reply:
I'm not sure. I just added for easy implementation of this PR. I can remove `Sync` and wrap the `dyn TransportSend` with `Mutex`. We have to revise the transport abstraction later anyway.

Reply:
I think adding `Sync` as dependency looks good for now.